ocrolib/common.py

```diff
@@ -665,34 +666,50 @@ def expand_args(args):
     else:
         return args
 
-data_paths = [
+
+_data_paths = [
     ".",
-    "./models",
-    "./data",
-    "./gui",
-    "/usr/local/share/ocropus/models",
-    "/usr/local/share/ocropus/data",
-    "/usr/local/share/ocropus/gui",
-    "/usr/local/share/ocropus",
+    "models",
+    "data",
+    "gui",
 ]
 
-def ocropus_find_file(fname,gz=1):
+
+def ocropus_find_file(fname, gz=True):
     """Search for OCRopus-related files in common OCRopus install
     directories (as well as the current directory)."""
     if os.path.exists(fname):
         return fname
     if gz:
-        if os.path.exists(fname+".gz"):
-            return fname+".gz"
-    for path in data_paths:
-        full = path+"/"+fname
-        if os.path.exists(full): return full
+        if os.path.exists(fname + ".gz"):
+            return fname + ".gz"
+
+    for path in _data_paths:
+        full = os.path.join(path, fname)
+        if os.path.exists(full):
+            return full
+    if gz:
+        for path in _data_paths:
+            full = os.path.join(path, fname + ".gz")
+            if os.path.exists(full):
+                return full
+
+    share_path = os.path.join(sysconfig.get_config_var("datarootdir"),
```

Comment:
Won't this depend on the location where Python was installed rather than where the module data was installed? For my box, this returns `/usr/share/ocropus` while `python setup.py install` installs the models to `/usr/local/share`. How about something like

```python
for prefix in ['.', '/usr/local/share/ocropus', '/usr/share/ocropus']:
  fpath = os.path.join(prefix, fname)
  if os.path.exists(fpath): return fpath
  if gz and os.path.exists(fpath+".gz"): return fpath + ".gz"
  for subdir in ["models", "data","gui"]:
    fpath = os.path.join(prefix, subdir, fname)
    if os.path.exists(fpath): return fpath
    if gz and os.path.exists(fpath+".gz"): return fpath + ".gz"